e2e-tests/cypress/integration/appeal-submission-site-health-and-safety-issues.feature

```diff
@@ -0,0 +1,60 @@
+  my site visit accordingly
+
+ # On page load none of the option should be selected, this will include the page navigation from task list
+ Scenario: Opportunity to provide health and safety issues is presented
+   Given the status of the appeal section is displayed
+   When health and safety issues is accessed
+   Then the opportunity to provide health and safety issues is presented
+
+ Scenario: Confirmation of health and safety issues not provided
+   Given confirmation of health and safety issues is requested
+   When no confirmation of health and safety issues is provided
+   Then appeal is not updated because confirmation of health and safety issues is required
+
+ Scenario: Confirmation provided of health and safety issues-Exceeds character limit
+   Given confirmation of health and safety issues is requested
+   When confirmation of health and safety issues is provided along with concerns exceeding the limit
+   Then appeal is not updated because the health and safety concerns exceed the limit
+
+ Scenario: Health and safety issues present but concerns not provided
+   Given confirmation of health and safety issues is requested
+   When confirmation of health and safety issues is provided without concerns
+   Then appeal is not updated because health and safety concerns are required
+
+ Scenario: Confirmation provided of health and safety concerns
+   Given confirmation of health and safety issues is requested
+   When confirmation of health and safety issues is provided along with concerns
+   Then appeal is updated with health and safety issues and concerns and the appeal tasks are presented
+
+ Scenario: Confirmation provided no health and safety issues
+   Given confirmation of health and safety issues is requested
+   When confirmation of no health and safety issues is provided
+   Then appeal is updated with no health and safety issues and the appeal tasks are presented
+
+ Scenario: Confirmation indicated with health and safety issues-update the appeal with no health and safety issues
+   Given health and safety issues along with concerns have been indicated
+   When confirmation of no health and safety issues is provided
+   Then appeal is updated with no health and safety issues and the appeal tasks are presented
+
+ Scenario: Confirmation indicated with no health and safety issues-update the appeal with health and safety issues
+   Given no health and safety concerns have been indicated
+   When confirmation of health and safety issues is provided along with concerns
+   Then appeal is updated with health and safety issues and concerns and the appeal tasks are presented
+
+# Indicated means not saved(save and continue button is not selected)
+ Scenario: Confirmation provided with no health and safety issues-replace the present selection of health and safety concerns
+   Given Health and safety issues and concerns previously indicated followed by an indication of no issues
```

Comment:
idk if this is viable or not, but could the "followed by..." be another step? 

feel free to ignore me!

Reply:
explained to Chris that these are two statements combined to one statement. Resolve